src/lib/create.ts

```diff
@@ -12,18 +12,16 @@ export async function create(
   this: Connection,
   query: string
 ): Promise<ContractReceipt> {
-  // TODO: depending on the outcome of some discussions this check to see if address is in
```

Comment:
As discussed internally we are no longer "gating" in this way.  There is a pending discussion of "attributing", but if that happend it will be a parameter in the SC call